core/src/components/range/test/states/range.e2e.ts

```diff
@@ -72,3 +72,27 @@ configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
     });
   });
 });
+
+configs({ modes: ['ionic-md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('range: focus'), () => {
```

Comment:
```suggestion
  test.describe(title('range: states'), () => {
```
Let's give this suite a generic name in case we want to add more state tests.